src/main/java/bio/terra/app/utils/SentryUtils.java

```diff
@@ -0,0 +1,46 @@
+package bio.terra.app.utils;
+
+import bio.terra.app.configuration.SentryConfiguration;
+import bio.terra.common.exception.BadRequestException;
+import bio.terra.common.exception.ConflictException;
+import bio.terra.common.exception.ForbiddenException;
+import bio.terra.common.exception.NotFoundException;
+import bio.terra.common.exception.NotImplementedException;
+import io.sentry.Sentry;
+import java.util.Optional;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.servlet.NoHandlerFoundException;
+
+public class SentryUtils {
+  private static final String DEFAULT_UNDEFINED_ENVIRONMENT = "undefined";
+
+  public static void initializeSentry(SentryConfiguration sentryConfiguration) {
+    String dsn = Optional.ofNullable(sentryConfiguration.getDsn()).orElse("");
+    String environment =
+        Optional.ofNullable(sentryConfiguration.getEnvironment())
+            .orElse(DEFAULT_UNDEFINED_ENVIRONMENT);
```

Comment:
There is no need to allocate an `Optional` here. You can instead use `Objects.requireNonNull()` which doesn't cause an object to be created.
```suggestion
    String dsn = Objects.requireNonNullElse(sentryConfiguration.dsn(), "");
    String environment =
        Objects.requireNonNullElse(sentryConfiguration.environment(), DEFAULT_UNDEFINED_ENVIRONMENT);
```

Reply:
Ok, great! Thanks!